.eslintrc.json

```diff
@@ -45,6 +45,7 @@
 				"local-rules/umb-class-prefix": "error",
 				"local-rules/prefer-static-styles-last": "warn",
 				"local-rules/ensure-relative-import-use-js-extension": "error",
+				"local-rules/ensure-no-external-imports": "error",
```

Comment:
can we call the rule something like "app no direct node_module imports", so the name doesn't collide with the naming of our "external"-module?

Reply:
Great idea! Here is what Github Copilot suggests, what do you think? 😅

> GitHub Copilot
You could consider naming the rule something like enforce-umbraco-external-imports. This name clearly communicates the purpose of the rule (to enforce imports from the @umbraco-cms/backoffice/external module) and should not conflict with the name of your external module.

Reply:
Brilliant 🎉 😄